Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -112,6 +112,14 @@ def bump_formula_pr
     odie "This formula is not in a tap!" if formula.tap.blank?
     odie "This formula's tap is not a Git repository!" unless formula.tap.git?
 
+    autobump_file_path = formula.tap.path/".github/autobump.txt"
+    if formula.tap.core_tap? && File.readlines(autobump_file_path).grep(/#{formula.name}/).any?
```

Comment:
```suggestion
    if formula.tap.core_tap? && File.readlines(autobump_file_path, chomp: true).include?(formula.name)
```

Same reason as above.

Reply:
```suggestion
    if formula.tap.core_tap? && autobump_file_path.exist? && autobump_file_path).readlines(chomp: true).include?(formula.name)
```